l1-contracts/src/core/Decoder.sol

```diff
@@ -341,25 +351,23 @@ contract Decoder {
     }
 
     bytes32 diffRoot = _computeRoot(baseLeafs);
-
+    bytes32[] memory l1ToL2Messages;
     bytes32 messagesHash;
     {
       uint256 messagesHashPreimageSize = 0x20 * L1_TO_L2_MESSAGES_PER_ROLLUP;
```

Comment:
nit: Should `messagesHashPreimageSize` be renamed to `L1ToL2MesssagesHashPreimageSize` for consistency?
Same for `messagesHash`. It is unclear to me if it is for L1toL2 or L2toL1 or does it also include other kind of messages. In the natspec, you call this `L1ToL2MessagesHash`